docs/api/utils/zmq_helpers.rst

```diff
@@ -1,5 +0,0 @@
-qcodes.utils.zmq_helpers
-------------------------
-
-.. automodule:: qcodes.utils.zmq_helpers
-   :members:
```

Comment:
Sorry you will also need to remove the ref to this from the index file in the same dir

Reply:
no, it's on me, should've done the `grep` homework :)